components/card/card.css

```diff
@@ -0,0 +1,109 @@
+:root{
+    --card-primary-color: #2a9d8f;
+    --card-font-color : #fff;
+    --card-border-color : #bacdd8;
+    --card-dark-color : #000;
+    --card-status-color : #807F7F;
+}
+.card-container {
+    max-width: 230px;
+    min-width: 220px;
+    margin: 0 auto;
+}
+.card-img {
+    width: 100%;
+    border-radius: 0.5em;
+    height: 12em;
+    object-fit: fill;
+}
+.card {
+position: relative;
+background-color: var(--card-font-color);
+border: 1px solid var(--card-border-color);
+padding: 4px;
+border-radius: 0.5em;
+}
+.card-details {
+padding: 0em 0.5em;
+}
+.name {
+color: var(--card-primary-color);
+font-weight: 600;
+line-height: 0.1em;
+}
+.btn {
+width: 100%;
+margin: 0.1em 0;
+cursor: pointer;
+}
+.shadow-card{
+    box-shadow: rgba(17, 17, 26, 0.1) 0px 1px 0px, rgba(17, 17, 26, 0.1) 0px 8px 24px, rgba(17, 17, 26, 0.1) 0px 16px 48px;
+}
+.card-below{
+    position: relative;
+}
+.card-badge p{
+    position: absolute;
+    top: 0;
+    right: 0;
+    background-color: var(--card-primary-color);
+    border-radius:  0.5em 0 0 0.5em ;
+    padding: 0em 1em;
+    color: var(--card-font-color);
+}
+.card-dismiss{
+    position: absolute;
+    top: -0.5em;
+    right: 0;
+    padding: 1em 1em;
+    color: var(--card-dark-color);
+}
+.btn-dismiss{
+    color: var(--card-primary-color);
+    cursor: pointer;
+}
+.text-striked{
+    text-decoration: line-through;
+    font-size: 0.8em;
+    color: var(--card-status-color);
+}
+
+.card-status{
+    font-size: 2rem;
```

Comment:
use only one unit like if you using rem then use it everywhere and rem is best practice for responsive design.

Reply:
done